src/Microsoft.ML.HalLearners/OlsLinearRegression.cs

```diff
@@ -34,25 +34,32 @@
 namespace Microsoft.ML.Trainers.HalLearners
 {
     /// <include file='doc.xml' path='doc/members/member[@name="OLS"]/*' />
+    [BestFriend]
     public sealed class OlsLinearRegressionTrainer : TrainerEstimatorBase<RegressionPredictionTransformer<OlsLinearRegressionModelParameters>, OlsLinearRegressionModelParameters>
     {
-        public sealed class Arguments : LearnerInputBaseWithWeight
+        public sealed class Options : LearnerInputBaseWithWeight
         {
             // Adding L2 regularization turns this into a form of ridge regression,
             // rather than, strictly speaking, ordinary least squares. But it is an
             // incredibly uesful thing to have around.
+            /// <summary>
+            /// L2 regularization weight.
+            /// </summary>
```

Comment:
Please merge them into summary.

Reply:
? the comments above? 

---
In reply to: [248857752](https://github.com/dotnet/machinelearning/pull/2170#discussion_r248857752) [](ancestors = 248857752)

Reply:
Yep.
```
            // Adding L2 regularization turns this into a form of ridge regression,
            // rather than, strictly speaking, ordinary least squares. But it is an
            // incredibly useful thing to have around.
```
looks more like a doc string.

---
In reply to: [248858191](https://github.com/dotnet/machinelearning/pull/2170#discussion_r248858191) [](ancestors = 248858191,248857752)